### DISCUSSION TITLE
Daily 18/12/2023

### DISCUSSION THREAD

**Discussion by lucasdpt** [Sinan-Karakaya/R-Type#67] [Daily]
Suite de l'implémentation réseau dans le serveur

**Comment by Sinan-Karakaya** [comment]
Harceler quelqu'un pour nous aider avec le Lua
Répondre aux questions des autres pour les détails finaux

**Comment by JasonLAGOUTE** [comment]
Suite de l'implémentation réseau dans le serveur

**Comment by mlbonniec** [comment]
**Fait**
* Compatibilité macOS
* UIRectangleShape

**À faire**
* Implémentation du réseau côté client
* Implémentation des components liés au réseau

**Comment by jovanHillion** [comment]
# ToDo:

- terminer l'implémentation du scripting lua, qui actuellement fonctionne -> (toute le journée)
